generator/graphql-kotlin-schema-generator/src/main/kotlin/com/expediagroup/graphql/generator/execution/PropertyDataFetcher.kt

```diff
@@ -25,7 +26,7 @@ import kotlin.reflect.KProperty
  *
  * @param propertyGetter Kotlin property getter that will be invoked to resolve a field
  */
-class PropertyDataFetcher(private val propertyGetter: KProperty.Getter<*>) : DataFetcher<Any?> {
+class PropertyDataFetcher(private val propertyGetter: KProperty.Getter<*>) : DataFetcher<Any?>, TrivialDataFetcher<Any?> {
```

Comment:
TrivialDataFetcher<T> already implements DataFetcher<T>, should we just 
`class PropertyDataFetcher(private val propertyGetter: KProperty.Getter<*>) : TrivialDataFetcher<Any?>` ?

Reply:
Good callout. 👍